fe-library-app/src/components/AppRouter/pages/HomePage.tsx

```diff
@@ -7,40 +7,54 @@ import SearchBar from '../../SearchBar/SearchBar'
 import BookList from '../../BookList/BookList'
 import { BookItemResponse, getBooks, GetBooksRequest } from '../../../services/BookService'
 
+export interface Filters{
+  description: string,
+  isbn: string,
+  authorFirstName: string,
+  authorLastName: string
+}
+
 const HomePage = () => {
   const [ bookList, setBookList ] = useState<BookItemResponse[]>([])
   const [ search, setSearch ] = useState('')
+  const [ filters, setFilters ] = useState<Filters>()
+  const [ orders, setOrders ] = useState<string[]>([])
   const [ pageNumber, setPageNumber ] = useState(1)
   const [ isResponseEmpty, setIsResponseEmpty ] = useState(true)
-
-  const getBooksPageAppend = (request: GetBooksRequest) => {
-    getBooks(request)
-      .then((response) => {
-        setBookList(prevList => [ ...prevList, ...response.data.Items ])
-        setIsResponseEmpty(request.PageNumber * request.PageLength >= response.data.TotalCount)
-      })
-      .catch(() => alert('Something went wrong!'))
-  }
+  const [ isThereBooks, setIsThereBooks ] = useState(false)
```

Comment:
Do you really need this additional state? 
You already have a bookList and you can use it. So, my suggestion is to use `bookList.length` for the condition.